src-docs/src/views/markdown_editor/markdown_format.js

```diff
@@ -4,7 +4,7 @@ import { EuiMarkdownFormat } from '../../../../src';
 
 const markdownContent = `Beyond Remark's base syntax, **EuiMarkdownFormat** bundles these abilities by default:
 
-\`:smile:\` we support emojis :smile:!
+\`:smile:\` we support emojis :smile:! But not x)
```

Comment:
`:)` might be a better/more universal example over `x)` - WDYT?
```suggestion
\`:smile:\` we support emojis :smile:! But not :)
```

Reply:
The funny thing is that you would think so, but all real world cases came from things like `(max)` with `x)`

Reply:
That's why I specifically added that one.

Reply:
Right yeah, that makes sense for production + for testing, but since this is documentation, I think it makes the most sense to use an example that directly correlates to `:smile:`. Also possible most people may not be familiar with `x)` as an emoticon (but will know `:)` since it's so ubiquitous).

Reply:
Fair enough